### PR TITLE
Convert speakers table to ember table and integrate Speakers API on it

### LINKED ISSUE #638: Integrate API at the events/<event_id>/speakers route.
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend
event-speakers-api

View, Edit, Delete actions to be dealt in separate issue.

### PR DESCRIPTION
**Pull request by ritikamotwani**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Converts the Speakers table into an ember model table and integrates it with the speakers API.

#### Changes proposed in this pull request:

- The speakers table is still not a proper ember table. This PR converts it into that and integrates it with the speakers API.
image

Fixes #638

### CODE REVIEW

File: app/templates/components/ui-table/cell/events/view/speakers/cell-simple-sessions.hbs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class="ui unordered list">
+  {{#each record as |session|}}
```
Comment (CosmicCoder96) [inline app/templates/components/ui-table/cell/events/view/speakers/cell-simple-sessions.hbs]: you can't refer to a related record like this, @ritikamotwani . use 
```hbs
 {{#each record.sessions as |session|}}
```
. As `sessions` are a related field.

**Comment by ritikamotwani** [comment]
@niranjan94 @CosmicCoder96 @geekyd could you please help with the build. I am unable to figure out the reason of its failure again and again.

File: app/routes/events/view/speakers/list.js [hunk 2]
```diff
@@ -1,57 +1,43 @@
...
+          name : 'state',
+          op   : 'eq',
+          val  : 'rejected'
+        }
+      ];
+    }  else {
+      filterOptions = [];
+    }
+    return this.modelFor('events.view').query('speakers', {
+      include      : 'sessions,event',
```
Comment (CosmicCoder96) [inline app/routes/events/view/speakers/list.js]: @ritikamotwani Why have you included `event`?

File: app/routes/events/view/speakers/index.js [hunk 3]
```diff
@@ -1,47 +1,11 @@
-import Route from '@ember/routing/route';
+import Ember from 'ember';
+
+const { Route } = Ember;
```
Comment (CosmicCoder96) [inline app/routes/events/view/speakers/index.js]: why?

File: app/routes/events/view/speakers/list.js [hunk 4]
```diff
@@ -1,57 +1,43 @@
-import Route from '@ember/routing/route';
```
Comment (CosmicCoder96) [inline app/routes/events/view/speakers/list.js]: why?

File: app/controllers/events/view/speakers/list.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
```
Comment (CosmicCoder96) [inline app/controllers/events/view/speakers/list.js]: instead of this use,
```js
import Controller from '@ember/controller';
```

**Comment by ritikamotwani** [comment]
@CosmicCoder96 @niranjan94 please review.

**Comment by ritikamotwani** [comment]
@niranjan94 please review.

**Comment by ritikamotwani** [comment]
@niranjan94 @pradeepgangwar @mayank8318 please review.

**Pull request merged**